src/ert/_c_wrappers/job_queue/ert_script.py

```diff
@@ -37,9 +36,6 @@ def stderrdata(self) -> str:
             self._stderrdata = self._stderrdata.decode()
         return self._stderrdata
 
-    def isVerbose(self):
-        return self.__verbose
```

Comment:
Could you split the first commit into "Clean up workflow verbose flag" and "refactor function_ert_script.py". I am having some problem seeing what goes where.